packages/block-editor/src/components/copy-handler/README.md

```diff
@@ -0,0 +1,42 @@
+# Copy Handler
+
+The `CopyHandler` component handles the copy, cut and paste events of its children blocks. It handles multiple selection of blocks as well.
+
+Concretely, it handles the display of success messages and takes care of copying the block to the clipboard using the [raw handling API](https://github.com/WordPress/gutenberg/tree/master/packages/blocks/src/api/raw-handling).
+
+![Copy cut behaviours](https://make.wordpress.org/core/files/2020/09/copy-cut-behaviour.gif)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Adds the copy handler functionnalities to the editor.
+
+```jsx
+import { BlockList, CopyHandler } from '@wordpress/block-editor';
+
+const MyCopyHandler = () => (
+	<CopyHandler>
+		<BlockList />
+	</CopyHandler>
+);
+```
+
+_Note:_ The `CopyHandler` only catches cut, copy and paste events coming from its props.children. In this example, the child is the `BlockList` component.
```

Comment:
`CopyHandler` also only intercepts such events when there is a block-editing context and at least one block is currently selected.